### ISSUE TITLE
Update Target API level to 34

### ISSUE THREAD

**Issue by Jakeler** [Jakeler/UT61E-Toolkit#22]
> From 31 Aug 2023, if your target API level is not within two years of the latest Android release, your app will not be available to new users running Android versions higher than your app's target API level.

Noticed here https://github.com/Jakeler/Jakeler.github.io/issues/7

**Comment by ZIMkaRU** [comment]
> You could still install it with the apk, but I am also working on fixing google play with an update.

taking into account that works in progress, would you be able to upload the last version of the APK to the release page section, please
(I see apk only for `v1.0`)

**Comment by Jakeler** [comment]
Yeah good idea, added now 1.4 apk: https://github.com/Jakeler/UT61E-Toolkit/releases/tag/v1.4

Sorry for the inconvenience, these SDK, java and gradle updates change quite a bit - didn't find the time to fix everything.

**Comment by ZIMkaRU** [comment]
anyway thanks for what you do

**Comment by Jakeler** [comment]
Finished and released as v2: https://github.com/Jakeler/UT61E-Toolkit/releases/tag/v2.0.0
Play Store update follows (at least for existing installs)
